### ISSUE TITLE
As a developer, I want to be able to sort the issues in the boards by update date

### ISSUE THREAD

**Issue by marcinsolarski** [ZenHubIO/support#1079]
Currently, zenhub supports sorting issues in the boards view by creation date but not by update date. With many issues being updated, I'd like to see the most recently updated on the top when reviewing the latest progress.

**Comment by kendramoroz** [comment]
@marcinsolarski thanks for the feedback here. Similar to https://github.com/ZenHubIO/support/issues/755, we've added this to the backlog of enhancements to consider for future updates. Bumping this over to the discussion pipeline. If you have any other feedback to add, feel free to drop us a line!

**Comment by bmunkholm** [comment]
Agree. I would even go so far as replacing the current sort by creation with sort by update. I have never needed sort by creation.
